sandy/zam.py

```diff
@@ -498,6 +627,8 @@ def zam2za(zam, method="nndc"):
 def z2sym(z):
     return ELEMENTS[z]
 
+def sym2z(sym):
+    return ATOMIC_NUMBERS[sym]
```

Comment:
Do we need a function? We already have a dictionary

Reply:
I did it to be consistent with the function `z2sym`, I will remove both

Reply:
Do you need them? If yes you keep them, else you remove them.
Maybe also `z2sym` should be removed.